libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.h

```diff
@@ -259,7 +259,7 @@ typedef NS_ENUM(NSInteger, SFOAuthCredentialsStorageType){
  @param type Indicates whether the OAuth credentials are stored in the keychain
  @return An initialized authentication credential object.
  */
-- (instancetype)initWithIdentifier:(NSString *)theIdentifier clientId:(NSString *)theClientId encrypted:(BOOL)encrypted storageType:(SFOAuthCredentialsStorageType)type;
+- (_Nullable instancetype)initWithIdentifier:(NSString * _Nullable )theIdentifier clientId:(NSString * _Nullable)theClientId encrypted:(BOOL)encrypted storageType:(SFOAuthCredentialsStorageType)type;
```

Comment:
Following the configuration through, `SFOAuthCredentials.identifier` is `nonnull`, which means `theIdentifier` should be too.